Add getting started section to readme (#397)
cytomining/CytoTable: readme.md

pip install git+https://github.com/cytomining/CytoTable.git
```

## Getting started

> Check out the following resources to get started!

Reviewer: ```suggestion
Check out the following resources to get started with CytoTable!
```

We created tutorials which follow a narrative-driven approach.

> There are also Jupyter notebooks for pragmatic, hand-on explanations of working with CytoTable.

Reviewer: ```suggestion
We also provide Jupyter notebooks for pragmatic, hands-on explanations.
```
